### PR TITLE
New release notes (v.7) - Beta

### PR DESCRIPTION
**Pull request by fpetru**
Includes initial version of the release notes.

### CODE REVIEW

File: docs/releases.md [hunk 1]
```diff
@@ -1,5 +1,56 @@
...
+- **DietPi-Software** | **PaperMC**  :octicons-arrow-right-16:  Resolved an issue where the install failed due to changed download URLs and stabilised service start and config creation by setting the Java heap size and allowing more time for the startup on smaller SBCs. Many thanks to @omavoss for reporting this issue: https://dietpi.com/phpbb/viewtopic.php?p=30191#p30191
+- **DietPi-Software** | **OpenTyrian**  :octicons-arrow-right-16: The install option has been disabled on x86_64, since the Debian package comes with a different file structure and it has been disabled on 64-bit RPi systems, since the binary has been compiled for armhf.
+- **DietPi-Software** | **Domoticz** :octicons-arrow-right-16: Resolved an issue where saving custom scripts and starting with a template did not work. Many thanks to @tec13 for reporting this issue: https://dietpi.com/phpbb/viewtopic.php?t=8627
+- **DietPi-Software**  :octicons-arrow-right-16: Resolved an issue where for ruTorrent, Koel and Bitwarden_RS the automatic newest version detection failed and instead a possibly older fallback was used. Many thanks to @kelvmod for reporting this issue: https://github.com/MichaIng/DietPi/issues/4105
```
Comment (MichaIng) [inline docs/releases.md:40]: One additional bug fix:
```
- **DietPi-Software** | **LXQt** :octicons-arrow-right-16: Resolved visual issues with our default configuration of Debian Buster, drastically simplified and cleaned up the files we ship.
```

Comment (StephanStS) [inline docs/releases.md:40]: "ruTorrent" -> "rTorrent/ruTorrent".

The package description in software/bittorrent is called 
"rTorrent - BitTorrent server with ruTorrent web interface"

File: docs/releases.md [hunk 2]
```diff
@@ -1,5 +1,56 @@
...
+
+As always, many smaller code performance and stability improvements, visual and spelling fixes have been done, too much to list all of them here. Check out all code changes of this release on GitHub: https://github.com/MichaIng/DietPi/pull/4XXX
+
```
Comment (StephanStS) [inline docs/releases.md:43]: Ist the link with pull/4XXX correct?

File: docs/releases.md (hunk 1 above)
Comment (MichaIng) [reply docs/releases.md:40]: But the automatic version detection is done for the ruTorrent archive indeed, while rTorrent is simply installed from the APT repository. So the log is correct 😉.

File: docs/releases.md (hunk 2 above)
Comment (MichaIng) [reply docs/releases.md:43]: Ah, there does not exist a PR yet. I'll create one, so we can fix this. It's independent from merge order, so can be safely done even if we ship more beta releases.

Comment (fpetru) [reply docs/releases.md:43]: Super. I'll do all the changes. 
The PR is still draft.

**Comment by MichaIng** [comment]
@fpetru 
Shall we add it already for the beta now or do you want to finish it when the release is done?

**Comment by fpetru** [comment]
Good suggestion @MichaIng - I've merged this PR to be available in this week release. I will continue to work, expanding the documentation for the new software titles.

**Pull request merged**